frontend/packages/dev-console/integration-tests/features/operators.feature

```diff
@@ -83,16 +83,28 @@ Scenario: Install OpenShift Virtualization Operator: VM-01-TC01
 
 
 @regression, @smoke
-Scenario: Create HyperConverged Cluster: VM-01-TC02
+Scenario: Create OpenShift Virtuliaztion Deployment
     Given user has selected namespace "openshift-cnv"
```

Comment:
namespace "openshift-cnv' cannot be created, so you can update it as "user installed virtualization operator"

Reply:
updated